charts/paperless/values.yaml

```diff
@@ -140,6 +140,24 @@ consumption:
   # -- Temporary emptyDir volume for the consumption directory. Only used if type equals `emptyDir` or is unknown.
   emptyDir: {}
 
+media:
+  # -- Enable the volume mount of a [media directory](https://docs.paperless-ngx.com/configuration/#paths-and-folders).
+  enabled: true
```

Comment:
In my opinion it would be great to keep this feature opt-in and set the `enabled` value to `false` as default.